nl.json

```
@@ -3717,13 +3717,13 @@
         "key": "21111B074FED432D86A1CCAE77CC81C3",
         "location": "/Game/UI/waiting.waiting_C:GetText [Script Bytecode]",
         "id": "Loading...",
-        "str": "Laden..."
+        "str": "Bezig met laden..."
     },
     {
         "key": "54288BFC4059AA21CBC614AA74C19EBE",
         "location": "/Game/UI/waiting.waiting_C:GetText [Script Bytecode]",
         "id": "Waiting players {players}",
-        "str": "Wachtende spelers {players}"
+        "str": "Wachtende spelers: {players}"
```

In this context is Waiting players (1/2) so the 2 dots is unnecesary
```suggestion
        "str": "Wachtende spelers {players}"
```